sketch_oct9a.ino

```diff
@@ -274,14 +274,14 @@ void setup() {
   // Pin D7: sensor LEDs
   pinMode(7, PinMode::OUTPUT);
 
-#if CAPSULE == 2
   pinPeripheral(0, PIO_SERCOM);
   pinPeripheral(1, PIO_SERCOM);
-#endif
   pinPeripheral(2, PIO_SERCOM);
   pinPeripheral(3, PIO_SERCOM);
+#if CAPSULE == 2
```

Comment:
I'm assuming this is where the temp/humid module is and sercom 0 and 1 are for the alt?

Reply:
Yep. When I moved the sensors around in the other PR I forgot to change this part. It still worked fine for capsule 2.